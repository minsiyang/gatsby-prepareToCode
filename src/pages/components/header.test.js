import React from "react"
import renderer from "react-test-renderer"
import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react';
import Header from "./header"
// import { render } from "react-dom";

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  });

  it("Renders prepare to code title" , () => {
    render(<Header/>)
    const message = "Prepare to code"
    expect(screen.getByText(message)).toBeInTheDocument();
  });

})