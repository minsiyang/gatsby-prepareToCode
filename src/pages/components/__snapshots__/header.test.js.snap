// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <h1>
     Prepare to code 
  </h1>
</div>
`;
